src/website/shared/fetchers.py

```diff
@@ -87,10 +87,43 @@ def make_problem_type(data: dict[str, Any]) -> models.ProblemType:
     return obj
 
 
+def to_camel_case(name: str) -> str:
+    """
+    >>> to_camel_case("abc_def")
+    abcDef
+    """
+    from re import sub
+
+    s = sub(r"(_|-)+", " ", name).title().replace(" ", "")
+    return "".join([s[0].lower(), s[1:]])
+
+
 def make_metric(data: dict[str, Any]) -> models.Metric:
     ctx: dict[str, Any] = dict()
     ctx["format"] = "cvssV3_1"
-    ctx["content"] = data.get("cvssV3_1", {})
+    raw_cvss = data.get("cvssV3_1", {})
+    ctx["raw_cvss_json"] = raw_cvss
+
+    if raw_cvss:
+        ctx["scope"] = raw_cvss.get("scope")
+        ctx["vector_string"] = raw_cvss.get("vectorString")
```

Comment:
Why do we handle these two separately? Currently they could be read in the loop with like the others.

Reply:
`vectorString` actually serves to generate the next fields as well. `scope` could be threaded through in the same way but these fields have different "statuses" in the specification.

Reply:
`attackVector` also has different choices, this is why I'm asking

Reply:
`attackVector` is derivable from `vectorString` though. This is the common point of all `vector_fields`, they are *fields* from the `vectorString`, it's not really the case of `baseScore`, etc.

Reply:
Ah, now I also understand what the TODO means. You want to check that the separately stored fields actually correspond what's in the `vectorString`.